Dockerfile.gpu

```diff
@@ -27,23 +27,22 @@ RUN echo 'export PATH=/opt/conda/bin:$PATH' > /etc/profile.d/conda.sh && \
 ENV PATH /opt/conda/bin:$PATH
 
 # Setup AllenNLP
-ENTRYPOINT ["/bin/bash", "-c"]
-
 WORKDIR /stage
 
-RUN conda create -n runenv --yes python=3.5
+RUN conda create -n allennlp --yes python=3.5
 ENV PYTHONHASHSEED 2157
 
 COPY . .
 
 # Get rid of pycache or .pyc files that you might have accidentally copied
 RUN find . | grep -E "(__pycache__|\.pyc|\.pyo$)" | xargs rm -rf
 
-RUN /bin/bash -c 'source activate runenv && scripts/install_requirements.sh &&\
- pip install --no-cache-dir -q http://download.pytorch.org/whl/cu80/torch-0.1.12.post2-cp35-cp35m-linux_x86_64.whl'
+RUN /bin/bash -c 'source activate allennlp && INSTALL_TEST_REQUIREMENTS="true" scripts/install_requirements.sh'
+RUN /bin/bash -c 'source activate allennlp && pip install --no-cache-dir -q http://download.pytorch.org/whl/cu80/torch-0.1.12.post2-cp35-cp35m-linux_x86_64.whl'
```

Comment:
I split this into its own line because it was timing out sometimes, and I didn't want to have to repeat the previous command.  I can recombine these if there is a clear advantage.